cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -740,9 +740,9 @@ func init() {
 	Root.AddCommand(endDevicesCommand)
 }
 
-var errAddressMismatch = errors.DefineAborted("address_mismatch", "server address mismatch")
+var errAddressMismatchEndDevice = errors.DefineAborted("end_device_address_mismatch", "network/application/join server address mismatch")
```

Comment:
The error ID should be `end_device_server_address_mismatch` (it's not about the end device address).

Reply:
corrected